Stage 3/script1.js

```diff
@@ -0,0 +1,147 @@
+const questionsArray = [];
+const player1 = {
+  name: "",
+  score: 0,
+  input: document.getElementById("scoreInput1"),
+  scoreDisplay: document.getElementById("score1"),
+};
+const player2 = {
+  name: "",
+  score: 0,
+  input: document.getElementById("scoreInput2"),
+  scoreDisplay: document.getElementById("score2"),
+};
+
+function randomizeOptions() {
+  const options = document.querySelectorAll('[name="options"]');
+  const optionsArray = Array.from(options);
+
+  optionsArray.sort(() => Math.random() - 0.5);
+
+  optionsArray.forEach((option, index) => {
+    option.value = `Random Option ${index + 1}`;
+  });
+}
+
+function saveQuestion() {
+  const question = {
+    question: document.getElementById("question").value,
+    options: Array.from(document.querySelectorAll('[name="options"]')).map(
+      (option) => option.value
+    ),
+    correctOption: document.querySelector('[name="correctOption"]:checked')
+      ?.value,
+  };
+
+  question.push(question);
+  renderQuestionList();
+}
+
+function addQuestion() {
+  const question = document.getElementById("question").value;
+  const option1 = document.getElementById("option1").value;
+  const option2 = document.getElementById("option2").value;
+  const option3 = document.getElementById("option3").value;
+  const option4 = document.getElementById("option4").value;
+  const correctOption = parseInt(
+    document.getElementById("correctOption").value
+  );
+
+  const newQuestion = {
+    question,
+    options: [option1, option2, option3, option4],
+    correctOption,
+  };
+
+  questionsArray.push(newQuestion);
+  updateQuestionList();
+  document.getElementById("questionForm").reset();
+}
+
+function updateQuestionList() {
+  const questionList = document.getElementById("questionList");
+  questionList.innerHTML = "";
+
+  questionsArray.forEach((q, index) => {
+    const listItem = document.createElement("li");
+    listItem.textContent = `Q${index + 1}: ${q.question}`;
+    questionList.appendChild(listItem);
+  });
+}
+
+function startQuiz() {
```

Comment:
This one has to be reworked, so that the player scores and corrext/wrong buttons are hidden in the beginning, and then shown when the quiz is startet..